ui/page/components/bottom_nav.go

```diff
@@ -52,7 +52,7 @@ func (bottomNavigationbar *BottomNavigationBar) LayoutBottomNavigationBar(gtx la
 				layout.Rigid(func(gtx C) D {
 					list := layout.List{Axis: layout.Horizontal}
 					return list.Layout(gtx, len(bottomNavigationbar.BottomNaigationItems), func(gtx C, i int) D {
-						appWidth := gtx.Dp(bottomNavigationbar.Load.CurrentAppWidth())
+						appWidth := gtx.Dp(unit.Dp(bottomNavigationbar.Load.CurrentAppWidth))
```

Comment:
in load file you noted the CurrentAppWidth not inited, so why you change it from CurrentAppWidth() to CurrentAppWidth?

Reply:
Dang, this and the change below came from rebasing this PR. I'll take a closer look.